.md/__tests__/scentific-reserch-GoT.test.js

```
@@ -0,0 +1,648 @@
+      const seasonData = [
+        { season: 1, deaths: 5, totalCharacters: 50 },
+        { season: 2, deaths: 8, totalCharacters: 55 },
+        { season: 3, deaths: 12, totalCharacters: 60 }
+      ];
+      
+      seasonData.forEach(season => {
+        const deathRate = season.deaths / season.totalCharacters;
+        expect(deathRate).toBeGreaterThan(0);
+        expect(deathRate).toBeLessThan(1);
+        expect(typeof deathRate).toBe('number');
+        expect(isFinite(deathRate)).toBe(true);
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)